SPECS-EXTENDED/iio-sensor-proxy/iio-sensor-proxy.spec

```diff
@@ -13,7 +13,6 @@ BuildRequires:  %{_bindir}/xsltproc
 BuildRequires:  make
 BuildRequires:  gcc
 BuildRequires:  pkgconfig(udev)
```

Comment:
This might also need to be removed: BuildRequires: pkgconfig(udev)

Pipeline shows error:
`
INFO[0463][graphpkgfetcher] Cache progress 52%: choosing 'systemd-bootstrap-devel-250.3-12.cm2.x86_64.rpm' to provide 'pkgconfig(udev)'. 
...
`
`
WARN[0024][scheduler] 1. package systemd-bootstrap-devel-250.3-12.cm2.x86_64 requires systemd-bootstrap = 250.3-12.cm2, but none of the providers can be installed 
...
`
`
ERRO[0024][scheduler] Failed to build iio-sensor-proxy-3.0-5.cm2.src.rpm, error: exit status 2, for details see: /mnt/vss/_work/1/s/build/logs/pkggen/rpmbuilding/iio-sensor-proxy-3.0-5.cm2.src.rpm.log 
`